app/controllers/personal-details.ts

```diff
@@ -53,7 +58,9 @@ function postDateOfBirth(updateAppealService: UpdateAppealService) {
 
 function getNamePage(req: Request, res: Response, next: NextFunction) {
   try {
-    const { personalDetails } = req.session.appeal.application || null;
+    const personalDetails = req.session.appeal.application.personalDetails && (req.session.appeal.application.personalDetails.givenNames || req.session.appeal.application.personalDetails.familyName) ?
+      req.session.appeal.application.personalDetails : { givenNames: req.idam.userDetails.forename, familyName: req.idam.userDetails.surname };
+
```

Comment:
Can we please move this logic to loadAppeal in update-appeal-service?

Reply:
Done